newspack-theme/classes/class-newspack-svg-icons.php

```diff
@@ -245,6 +245,20 @@ public static function get_social_link_svg( $uri, $size ) {
 		'mail'        => array(
 			'mailto:',
 		),
+		'mastodon'    => array(
+			// Regex pattern to match any .tld for the mastodon host name.
+			'#https?:\/\/(www\.)?mastodon\.(\w+)(\.\w+)?#',
```

Comment:
This didn't work for me when a `http://mastodon.lol/@user` link was supplied. But it did with this change:

```suggestion
			'mastodon.',
```

(same goes for the line below)

Reply:
🤦‍♀️ Thanks @adekbadek! I changed too many things in one go without retesting everything. This should now be resolved in [9a596b6](https://github.com/Automattic/newspack-theme/pull/2017/commits/9a596b69917e7be8ef57e12587b8d50fb53501c7)!